oneview_redfish_toolkit/app.py

```diff
@@ -231,23 +232,34 @@ def not_implemented(error):
             mimetype='application/json')
 
     config = util.config
-    if config["ssl"]["SSLType"] in ("self-signed", "adhoc"):
-        logging.warning("Server is starting with a self-signed certificate.")
-    if config["ssl"]["SSLType"] == "disabled":
-        logging.warning(
-            "Server is starting in HTTP mode. This is an insecure mode. "
-            "Running the server with HTTPS enabled is highly recommended.")
+
+    try:
+        host = config["redfish"]["host"]
+
+        # Gets the correct IP type based on the string
+        ipaddress.ip_address(host)
+    except ValueError:
+        logging.error("Host must be a valid IP address.")
```

Comment:
IP "informed ip" is not valid. Check the variable host on your config file.
is clear IMO

Reply:
Fixed!